src/test/java/examples/FirstClass.java

```diff
@@ -0,0 +1,14 @@
+package examples;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class FirstClass {
+    public static void main(String[] args) throws InterruptedException {
```

Comment:
Daca nu folosesti in alta parte nu cred ca are rost...

Reply:
Aici am facut la curs, si l-am lasat asa, trebuie sa il sterg? E doar ca primul exemplu.

Reply:
Da, e oki sa-l stergi, ce nu folosesti, sterge te rog